src/backend/distributed/sql/udfs/check_connection_to_node/latest.sql

```diff
@@ -0,0 +1,20 @@
+CREATE FUNCTION pg_catalog.check_connection_to_node (
```

Comment:
wish we had a well-defined naming policy to answer this, but: should this start with citus_ like other operational functions?

Reply:
I renamed to `citus_check_connection_to_node`

Reply:
historically, I renamed this function as follows:
`worker_check_connection_to_node` -> `check_connection_to_node` -> `citus_check_connection_to_node`

I think `citus_check_connection_to_node` is the best one so far.